src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/SuperFinalizeCheck.xml

```diff
@@ -15,6 +15,29 @@
  &lt;a href="https://javarevisited.blogspot.com/2012/03/finalize-method-in-java-tutorial.html"&gt;
  10 points on finalize method in Java&lt;/a&gt;.
  &lt;/p&gt;</description>
+          <properties>
+            <property default-value="^SuperFinalize.+$"
+                      name="format"
```

Comment:
I am not sure how you get all that properties, This Check has no properties at all.